paper-item.html

```diff
@@ -88,14 +88,14 @@
     <style include="paper-item-shared-styles"></style>
     <style>
       :host {
-        @apply(--layout-horizontal);
-        @apply(--layout-center);
-        @apply(--paper-font-subhead);
+        @apply --layout-horizontal;
+        @apply --layout-center;
+        @apply --paper-font-subhead;
 
-        @apply(--paper-item);
+        @apply --paper-item;
       }
     </style>
-
+    <slot></slot>
     <content></content>
```

Comment:
@e111077 Can you remove the `<content>` as well? `paper-item` has some shared styles on the class `.paper-item`, which unfortunately clashes with how Shady DOM scopes styles when extraneous elements (like `<content>`) are present.